test/reference.html

```diff
@@ -0,0 +1,946 @@
+    "4.4", "Matrices", "col1", 
+    "4.5", "Math spacing commands", "col2",
+    "4.6", "Dots", "col1",
+    "4.6.note", "", "note",
+    "4.7", "Nonbreaking dashes", "col1",
+    "4.8", "Roots", "col1",
+    "4.9", "Boxed formulas", "col1", 
+
+    "5", "Fractions and related constructions", "", 
+    "5.1", "Fractions", "col3",
+    "5.1.example", "", "", 
+    "5.2", "Binomials", "col3",
+    "5.2.example", "", "",
+    "5.3", "<kbd>\\genfrac</kbd>", "col1",
+    "5.4", "Continued fractions", "col1",
+
+    "6", "Delimiters", "col1",
+    "6.1", "Delimiter sizes", "col1",
+    "6.1.note", "", "note",
+    
+    "6.2", "Vertical bar notation", "",
+    "6.2.note", "", "note",
+    
+    "7", "The <kbd>\\text</kbd> command", "",
+    "7.1", "<kbd>\\mod</kbd> and its relatives", "col2",
+    "7.1.example", "", "",
+    
+    "8", "Integrals and sums", "",
+    "8.1", "Altering the placement of limits", "",
+    "8.2", "Multiple integral signs", "",
+    "8.3", "Multiple subscripts", "",
+    "8.4", "The <kbd>\\sideset</kbd> command", "",
+    
+];
+    var t = ''; // the output text
+    var i = 0;
+    var j = 0;
+    var matching = 0;
+    while (i < sections.length) {
+        if (sections[i + 1].length > 0) {
+            // Sections don't always have names, but if they do, use it as the header
+            var level = (sections[i].match(/\./g) || []).length + 1;
+            var sectionNumber = /\d$/.test(sections[i]) ? sections[i] + '&nbsp;' : '';
+            t += '<h' + level + '>' + sectionNumber + sections[i + 1] + '</h' + level + '>';
+        }
+        
+        // Open a section and use the class name specified as the last entry in the sections record
+        t += '<div class="' + sections[i + 2] + '">';
+        
+        j = 0;
+        if (sections[i].endsWith('.note')) {
+            // If the section number ends with ".note, simply display its content
+            while (j < entries.length) {
+                if (entries[j] === sections[i]) {
+                    t += '<p>' + entries[j + 1] + '</p>';
+                }
+                j += 2;
+            }
+        } else {
+            t += '<table>';
+            while (j < entries.length) {
+                if (entries[j] === sections[i]) {
+                    // entry matches the current section
+                    t += '<tr>';
+                    t += '<td class="mathquill-static-math">' + entries[j + 1] + '</td>';
+                    t += '<td><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
+                    t += '<td><code>' + entries[j + 1] + '</code></td>';
+                    t += '</tr>';
+                }
+                j += 2;
+            }
+            t += '</table>';
+        }
+        
+        t += '</div>';
+        i += 3;
+    };
+    
+    $('#table').html(t);
+    $('.mathquill-static-math').each(function() { 
+        MQ.StaticMath(this);
+        if (this.textContent === '$$\u200B') {
```

Comment:
this is very dependent on implementation details, I'd recommend using the API, like `MQ(this).latex() === ''`